code/loginscreen.py

```diff
@@ -49,19 +50,27 @@ def create_user(self):
         self.username = self.screen.getstr()
         self.screen_manager.display_mid('Please enter your desired Password: ')
         self.password = self.screen.getstr()
-        curses.endwin()
 
-        to_query = "SELECT * FROM users WHERE un LIKE \'%s\'" % self.username
-        rows = query.query(to_query, 'users', self.screen)
+        to_query = "SELECT * FROM pg_catalog.pg_roles WHERE rolname LIKE \'%s\'" % self.username
+        rows = query.query(to_query, 'postgres', self.screen)
         
         if rows:
```

Comment:
This seems to always pass when trying to create a username.  I printed out the contents of rows and it is ['rolname', 'rolsuper', 'rolinherit', 'rolcreaterole', 'rolcreatedb', 'rolcatupdate', 'rolcanlogin', 'rolreplication', 'rolconnlimit', 'rolpassword', 'rolvaliduntil', 'rolconfig', 'oid'][]

Have not been able to create a username because it always hits this if block and states username already exists.